src/main/webapp/index.js

```diff
@@ -0,0 +1,82 @@
+const firebaseApp = firebase.initializeApp({
+    apiKey: "AIzaSyAv-Zy1ZpAwC31yVhtcSgGIj3OmTUbkE5Y",
+    authDomain: "login-with-firebase-data-b3557.firebaseapp.com",
+    databaseURL: "https://login-with-firebase-data-b3557-default-rtdb.firebaseio.com",
+    projectId: "login-with-firebase-data-b3557",
+    storageBucket: "login-with-firebase-data-b3557.appspot.com",
+    messagingSenderId: "155976060237",
+    appId: "1:155976060237:web:7f8d2e6eab1f1895f93b04"
+}); //should change to team's firebase! currently juhee's own initializations.
+
+const db = firebaseApp.firestore();
+const auth = firebaseApp.auth();
+
+const register = () => {
+    const fullname = document.getElementById('fullname').value;
+    const email = document.getElementById('email').value;
+    const password = document.getElementById('password').value;
+    console.log(email, password);
```

Comment:
please remove this, we shouldn't print out the password

Reply:
got it—I was testing my code and forgot to remove these stuff!